src/Handlers/BookingsStateStatusesHandler.php

```diff
@@ -252,4 +326,19 @@ protected function _getVisibleStatuses($statuses)
             'statuses' => $statuses,
         ])->getParam('statuses');
     }
+
+    /**
+     * Retrieve user option.
+     *
+     * @since [*next-version*]
+     *
+     * @param int    $userId User ID.
```

Comment:
This should have been accepted also as a `float`, `string`, or `Stringable`, and [normalized to `int`].

[`NormalizeIntCapableTrait#_normalizeInt()`]: https://github.com/Dhii/normalization-helper-base/blob/develop/src/NormalizeIntCapableTrait.php#L30